Libraries/ActionSheetIOS/NativeActionSheetManager.js

```diff
@@ -25,6 +25,8 @@ export interface Spec extends TurboModule {
       +anchor?: ?number,
       +tintColor?: ?number,
       +userInterfaceStyle?: ?string,
+      +icons: ?Array<Object>, // TODO: This should be a ResolvedAssetSource
+      +tintIcons: boolean,
```

Comment:
```suggestion
      +icons?: ?Array<Object>, // TODO: This should be a ResolvedAssetSource
      +tintIcons?: boolean,
```

Reply:
As noted in the PR description, I chose these option names to match the existing ones from the Expo action sheet library, which uses ActionSheetIOS internally. The options are named this way for Android and web and changing them would introduce some complexity and incompatabilites. Details here: https://github.com/expo/react-native-action-sheet#androidweb-only-props